docs/usage/configuration-options.md

```diff
@@ -450,6 +450,71 @@ Make sure that version is a pinned version of course, as otherwise it won't be v
 
 For `sbt` note that Renovate will update the version string only for packages that have the version string in their project's `built.sbt` file.
 
+## bumpVersions
+
+Use this option to configure Renovate to increase a semantic version in your code when it updates dependencies.
+This is useful if Renovate does not support bumping the version in a package file which Renovate does not support, or the version is not in a package file.
```

Comment:
"Renovate does not support" is twice in one sentence. I think the sentence should be rephrased.